api/ip-api.yaml

```diff
@@ -0,0 +1,280 @@
+        '202':
+          description: All information package binaries deleted for this user
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/packageId'
+              examples:
+                packageIds:
+                  value: |-
+                    [
+                      {
+                          “uid”: “845872bf-cbf8-48e6-b495-ec6b4a63b607”,
+                          “name”: “IP.100.XYZ”,
+                      },
+                      {
+                          “uid”: “845872bf-cbf8-48e6-b495-ec6b4a63b609”,
+                          “name”: “IP.101.XYZ”,
+                      }
+                    ]
+  /ips/{packageId}:
+    get:
+      tags:
+        - /ips
+      description: Get the properties of an information package id.
+      summary: Get package info.
+      operationId: getIpById
+      parameters:
+        - in: path
+          name: packageId
+          description: UUID of the package to retrieve
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Full set of package properties
+          content:
+            application/json:
+              schema:
+                type: object
+                $ref: '#/components/schemas/informationPackage'
+              examples:
+                package:
+                  value: |-
+                    {
+                      “uid”: “845872bf-cbf8-48e6-b495-ec6b4a63b609”,
+                      “name”: “IP.101.XYZ”,
+                    }
+    delete:
+      tags:
+        - /ips
+      description: Delete information package binary by id.
+      summary: Delete IP by id.
+      operationId: deleteIpById
+      parameters:
+        - in: path
+          name: packageId
+          description: UUID of the package to delete
+          required: true
+          schema:
+            type: string
+      responses:
+        '201':
+           description: Package deleted
+           content:
+             application/json:
+               schema:
+                 type: object
+                 $ref: '#/components/schemas/packageId'
+               examples:
+                packageId:
+                  value: |-
+                    {
+                      “uid”: “845872bf-cbf8-48e6-b495-ec6b4a63b609”,
+                      “name”: “IP.101.XYZ”,
+                    }
+components:
+  schemas:
+    packageId:
+      type: object
+      properties:
+        uid:
+          type: string
+          nullable: false
+          format: uuid
+        name:
+          type: string
+          nullable: false
+    packageDetails:
+      type: object
+      properties:
+        id:
+          type: object
+          $ref: '#/components/schemas/packageId'
+        checksums:
+          type: array
+          items:
+            $ref: '#/components/schemas/digest'
+    digest:
```

Comment:
Should we go with digest or use the same terminology as METS? I.e. to call it checksum, that will have a value and a type (MD5, SHA1, etc.)

Reply:
Replace use of digest with checksum.